compiler/vmops.nim

```diff
@@ -304,3 +306,36 @@ proc registerAdditionalOps*(c: PCtx) =
     let fn = getNode(a, 0)
     setResult(a, (fn.typ != nil and tfNoSideEffect in fn.typ.flags) or
                  (fn.kind == nkSym and fn.sym.kind == skFunc))
+
+  proc urandomImpl(a: VmArgs) =
```

Comment:
hide behind `  if vmopsDanger in c.config.features:` (that's the exact purpose of `vmopsDanger`)